gobblin-audit/src/main/java/gobblin/audit/values/policy/column/ProjectAllColumnProjectionPolicy.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2014-2016 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed
+ * under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied.
+ */
+package gobblin.audit.values.policy.column;
+
+import java.util.List;
+
+import org.apache.avro.generic.GenericRecord;
+
+import com.typesafe.config.Config;
+
+import gobblin.annotation.Alias;
+import gobblin.audit.values.auditor.ValueAuditRuntimeMetadata;
+
+/**
+ * An {@link AbstractColumnProjectionPolicy} that projects all columns/fields of the <code>inputRecord</code>
+ */
+@Alias(value = "ProjectAllColumnProjectionPolicy")
```

Comment:
If we go with my previous comment about limited scope for aliases, we can a much shorter alias like "all".